src/resources/views/crud/fields/checklist.blade.php

```diff
@@ -25,7 +25,7 @@
     <label>{!! $field['label'] !!}</label>
     @include('crud::fields.inc.translatable_icon')
 
-    <input type="hidden" value="@json($field['value'])" name="{{ $field['name'] }}">
+    <input type="hidden" value='@json($field['value'])' name="{{ $field['name'] }}">
```

Comment:
I don't particularly like mixing apostrophes `'` and quotes `"` on html tags, but the only other way I see to solve the problem of having quotes `"` inside the attribute, is to use `value="{{ json_encode($field['value']) }}"`, which I think it's a worse solution because the rendered escaped code is a mess 😅
So I'll approve this way 👍

Reply:
True, the issue I experienced that strings will be set as: 
`value="["uuid"]"`
which breaks everything and results in a `]` being sent to the server.